src/python/py_tokenizer.cpp

```diff
@@ -19,8 +19,41 @@ namespace pyutils = ov::genai::pybind::utils;
 using ov::genai::ChatHistory;
 using ov::genai::TokenizedInputs;
 using ov::genai::Tokenizer;
+using ov::genai::PaddingMode;
+using PadModesVariant = std::variant<std::string, PaddingMode>;
+
+namespace {
+
+PaddingMode get_pad_mode(const PadModesVariant& pad_mode) {
+    PaddingMode result;
+    std::visit(pyutils::overloaded {
+    [&](PaddingMode padding_mode) {
+        result = padding_mode;
+    },
+    [&](std::basic_string<char> pad_mode_str) {
+        if (pad_mode_str == "truncate") {
```

Comment:
why do we need to allow `str` for this value?

Reply:
removed string. Left only enums.